app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/registerWithEoriAndIdModels.scala

```diff
@@ -227,6 +230,77 @@ case class RegisterWithEoriAndIdResponse(
 
 }
 
+trait CaseClassAuditHelper {
```

Comment:
Non-blocking question - is this needed? By doing a quick usage check it looks like it's not used, but I might be wrong.

Reply:
I'll delete it thanks..

Reply:
It's used in UpdateCustomsDataStoreConnector to serialise the dataStore request when making an audit call.